front/src/applications/stdcm/types.ts

```diff
@@ -185,11 +186,12 @@ export enum StdcmConfigErrorTypes {
   BOTH_POINT_SCHEDULED = 'bothPointAreScheduled',
   NO_SCHEDULED_POINT = 'noScheduledPoint',
   ZERO_LENGTH_PATH = 'zeroLengthPath',
+  MISSING_INFORMATIONS = 'missingInformations',
 }
 
 export type StdcmConfigErrors = {
   errorType: StdcmConfigErrorTypes;
-  errorDetails?: { originTime: string; destinationTime: string };
+  errorDetails?: { originTime?: string; destinationTime?: string; missingFields?: string[] };
```

Comment:
We should reinforce the `missingFields` type by creating a type : 
```
export type MissingField =
  | 'tractionEngine'
  | 'totalMass'
  | 'totalLength'
  | 'maxSpeed'
  | 'origin'
  | 'destination';
```
```suggestion
  errorDetails?: { originTime?: string; destinationTime?: string; missingFields?: MissingField[] };
```

Reply:
done